contracts/OwnableTwoSteps.sol

```diff
@@ -112,7 +112,7 @@ abstract contract OwnableTwoSteps is IOwnableTwoSteps {
         status = Status.RenouncementInProgress;
         _earliestOwnershipRenouncementTime = block.timestamp + delay;
 
-        emit InitiateOwnershipRenouncement();
+        emit InitiateOwnershipRenouncement(_earliestOwnershipRenouncementTime);
```

Comment:
Would it be cheaper if you don't read from storage, maybe cache `block.timestamp + delay` in memory?

Reply:
I did three versions:
- Caching in memory
- Recomputing a second time in the event
- Same

It looks like the current one is the cheapest and the second cheapest is using memory. Differences in gas are extremely small regardless.